README.md

```diff
@@ -105,15 +105,33 @@ For more information visit the [documentation](https://grafana.com/docs/k6-studi
 
 ---
 
-## Development
+## Contributing
 
-### Requirements
+If you're interested in contributing to the Grafana k6 Studio project:
 
-- Node.js >= v20
+- Start by reading the [Contributing guide](CONTRIBUTING.md)
+- Explore our [issues](https://github.com/grafana/k6-studio/issues) and see if there's anything you'd like to work on
+- Set up your development environment and start coding
 
-### Dev install
+### Development environment
+
+### Dependencies
+
+Make sure you have the following dependencies installed before setting up your developer environment:
+
+- [Git](https://git-scm.com/downloads)
+- [Node.js](https://nodejs.org/en/download/) >= v20
```

Comment:
Should we include chrome/chromium?

Reply:
technically is not mandatory if you are not using the recorder 🤔